Cargo.toml

```diff
@@ -18,8 +18,9 @@ keywords = ["metrics", "cloudwatch", "aws"]
 chrono = "0.4.13"
 futures-util = { version = "0.3.5", default-features = false }
 log = "0.4.11"
-metrics = { version = "0.12.1", features = ["std"] }
-rand = { version = "0.7", default-features = false, features = ["small_rng"] }
+metrics = { version = "0.13", features = ["std"] }
```

Comment:
This is a breaking change right? So we should bump metrics_cloudwatch to `0.11.0`?

Reply:
Yeah that will be done with `cargo release minor` after merging